src/fastoad/openmdao/exceptions.py

```diff
@@ -16,22 +14,37 @@
 #  You should have received a copy of the GNU General Public License
 #  along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+from os import PathLike
+from typing import Iterable
 
-from typing import List
-
+from fastoad._utils.files import as_path
 from fastoad.exceptions import FastError
 
 
-class FASTOpenMDAONanInInputFile(FastError):
+#  This file is part of FAST-OAD : A framework for rapid Overall Aircraft Design
+#  Copyright (C) 2024 ONERA & ISAE-SUPAERO
+#  FAST is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <https://www.gnu.org/licenses/>.
```

Comment:
The header/licence has been duplicated in the same script, it can be removed.

Reply:
Bad joke that PyCharm does from time to time :/ Fixed in 781349be84f21c6ced8c6769f2a251e49e8f49b3